src/gt4py/next/program_processors/runners/dace_iterator/itir_to_tasklet.py

```diff
@@ -534,7 +538,11 @@ def _visit_numeric_builtin(self, node: itir.FunCall) -> list[ValueExpr]:
         expr = fmt.format(*internals)
         node_type = self.node_types[id(node)]
         assert isinstance(node_type, itir_typing.Val)
-        type_ = itir_type_as_dace_type(node_type.dtype)
+        try:
+            type_ = itir_type_as_dace_type(node_type.dtype)
+        except NotImplementedError:
+            # return value of same type as operands, for example in max(a, b)
+            type_ = args[0].dtype
```

Comment:
This looks like it is just a work around for a problem in type inference?

Reply:
@tehrengruber

Reply:
We agreed on disabling the failing tests. These issues in type inference will be addressed by #1271